### ISSUE TITLE
Update /code command

### ISSUE THREAD

**Issue by twalton83** [TheOdinProject/odin-bot-v2#112]
We need to update the /code command to include a suggestion for Codepen or Repl.it for larger code snippets. This command can be found in the `botCommands/help.js`file.

*COMPLETION CRITERIA*:
- [ ] new copy including mention of one or both of the above services
- [x] snapshot tests are updated (see README for instructions) 
- [x] The automatic Discord embed has been removed from the link. See here: https://support.discord.com/hc/en-us/articles/206342858--How-do-I-disable-auto-embed-

**Comment by zainthedev** [comment]
Could I claim this one, please?

**Comment by zainthedev** [comment]
Looks like /code doesn't have it's own snapshot.
The test for it looks like this:
```
describe("/code snapshot", () => {
  // numbers 1, 3 & 10 are of no significance, they're just random quantities of mentions being used to test
  it.each([[1], [3], [10]])(
    "should return the correct output",
    (quantityOfMentions) => {
      const mentions = generateMentions(quantityOfMentions);
      expect(commands.code.regex.test(mentions)).toBeFalsy();
    }
  );
});
```
Looks like it doesn't mention a need for matching the snapshot. What should I do?
Considered separating it into it's own file (ie. outside of `help.js`) and expect a /code snapshot to be created. How does that sound?

**Comment by twalton83** [comment]
Closed by #115
